doc-content/enterprise-only/openshift/template-deploy-bcreplicas-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='template-deploy-bcreplicas-{context}-proc']
+= Configuring the {CENTRAL} replicas for a high-availability authoring environment
+
+If you are deploying a high-availability authoring environment, by default 2 replicas of {CENTRAL} are initially created. 
+
+Optionally, you can modify the number of replicas.
+
+Skip this procedure for a single authoring environment.
+
+.Prerequisites
+
+* You started the configuration of the template, as described in {startlink}.
+
+.Procedure
+
+To modify the number of initial replicas in a high-availability authoring environment, set the *KIE Server Container Replicas* (`KIE_SERVER_CONTAINER_REPLICAS`) parameter to the required number.
```

Comment:
This will modify the number of replicas of the Kie Server component, not the Business Central.
For Business Central is: *Business Central Container Replicas* (`BUSINESS_CENTRAL_CONTAINER_REPLICAS`) parameter.